sherpa/astro/ui/utils.py

```diff
@@ -6564,6 +6564,18 @@ def set_analysis(self, id, quantity=None, type='rate', factor=0):
         they are interpreted as the `id` and `quantity` parameters,
         respectively.
 
+        Sherpa does not match XSPEC when the `quantity` setting is set
+        to "channel", particularly when the data is grouped. Sherpa
+        retains the original CHANNEL values from the PHA dataset, so
+        calls to `ignore` and `notice` will use the same values even
+        if the grouping changes (although the results of these calls
+        may change depending on how the group edges match the selected
+        ranges), whereas XSPEC counts each group as a single channel.
```

Comment:
also here - I would remove the part of the sentence with the reference to XSPEC

Reply:
If we don't have the XSPEC reference then I'm not sure we should include this paragraph at all (discussed more above).